src/Azure.DataApiBuilder.Core/Authorization/AuthorizationResolver.cs

```diff
@@ -264,7 +260,7 @@ public void SetEntityPermissionMap(RuntimeConfig? runtimeConfig)
                     object[] Operations = permission.Operations;
                     foreach (JsonElement operationElement in Operations)
                     {
-                        Config.Operation operation = Config.Operation.None;
+                        Config.Operation operation =  Config.Operation.None;
```

Comment:
```suggestion
                        Config.Operation operation = Config.Operation.None;
```

Excess whitespace will cause the build to fail.